web-ts/src/components/FailureSwitcher.tsx

```diff
@@ -1,73 +1,71 @@
-// @flow
-
-import * as React from 'react'
+import React, { ReactNode } from 'react'
 import { NotFoundError } from 'api-client'
 import Failure from './Failure'
-import CategoriesRouteConfig from '../../app/route-configs/CategoriesRouteConfig'
-import EventsRouteConfig from '../../app/route-configs/EventsRouteConfig'
-import OffersRouteConfig from '../../app/route-configs/OffersRouteConfig'
-import PoisRouteConfig from '../../app/route-configs/PoisRouteConfig'
-import LocalNewsRouteConfig from '../../app/route-configs/LocalNewsRouteConfig'
-import TunewsRouteConfig from '../../app/route-configs/TunewsRouteConfig'
 import { LOCAL_NEWS_TYPE, TU_NEWS_TYPE } from 'api-client/src/routes'
 
-type PropsType = {|
+type PropsType = {
   error: Error
-|}
+}
 
 export class FailureSwitcher extends React.Component<PropsType> {
   /**
    * Renders a Failure with a link to the "home" of the route and information about what was not found
    * @param error
    * @return {*}
    */
-  static renderContentNotFoundComponent(error: NotFoundError): React.Node {
+  static renderContentNotFoundComponent(error: NotFoundError): ReactNode {
     const { city, language } = error
     switch (error.type) {
       case 'category':
         return (
           <Failure
-            goToPath={new CategoriesRouteConfig().getRoutePath({ city, language })}
+            // TODO use right redirect path
+            // goToPath={new CategoriesRouteConfig().getRoutePath({ city, language })}
```

Comment:
Is there a ticket for this?

Reply:
Not yet but we have to check all todos afterwards anyway.